packages/react-sdk/src/state/whiteboardManagerImpl.test.ts

```diff
@@ -72,9 +71,24 @@ describe('WhiteboardManagerImpl', () => {
     whiteboardManager.selectActiveWhiteboardInstance(event0, '@user-id');
     expect(destroySpy).not.toHaveBeenCalled();
 
-    whiteboardManager.selectActiveWhiteboardInstance(event1, '@user-id');
+    // Handler for the `Error: Channel not initialized` throw
+    const error = await new Promise<Error | null>((resolve) => {
+      process.once('uncaughtException', (err: Error) => {
+        resolve(err);
+      });
+
+      whiteboardManager.selectActiveWhiteboardInstance(event1, '@user-id');
+
+      expect(destroySpy).toHaveBeenCalled();
+
+      // After 250ms, the error should have been thrown. If not we resolve with null.
+      setTimeout(() => resolve(null), 250);
```

Comment:
Timeouts in tests are always not so great.
Do wo have alternatives here? e.g. a wait for call of `destroySpy`?

Reply:
Good question. Will check that.

Reply:
Seems to work